src/components/Clock/Clock.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Poppins:wght@400;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:wght@500&display=swap");
+.timer {
+  /* width: 100vw; */
+  height:25vh;
```

Comment:
use pixels here as vh here makes no sense
see the height and width of the box from the design